app/src/main/java/de/tum/in/tumcampusapp/fragments/CafeteriaDetailsSectionFragment.java

```diff
@@ -140,18 +136,18 @@ public void onClick(View view) {
                             String[] data = id.split("__");
                             String dishname = data[0];
                             int mensaId = Integer.parseInt(data[1]);
-                            AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
-                            Intent myIntent = new Intent(context, FavoriteDishReceiver.class);
+
+                            /*AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
```

Comment:
Sure, I just pushed it today so you have a basic idea on what I'm working on! I want to fix my comments in both pull requests this weekend.